quick_task/app/Models/department.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use DB;
+
+class department extends Model
+{
+    protected $table = 'departments';
+    protected $primaryKey = 'id';
+    protected $guarded = [];
+
+    //Thêm phòng 
+    public function insertDepart(){
+        // $data = [
+        //     'name' => ':name',
+        // ];
+        // DB::table('departments')->insert($data);
+        echo 'Thêm phòng ban';
+    }  
+
+    //Sửa tên phòng ban
+    public function updateDepart(){
+       //
+        echo 'Sửa tên phòng';
+    }
+
+    //Xóa phòng ban
+    public function deleteDepart(){
+       //
+        echo 'Xóa phòng ban';
+    }
+
+    //Lấy tất cả các dữ liệu của phòng ban
+    public function listDepart(){
+      //
+        echo 'Lấy tất cả các phòng';
+    }
+
+    //Phương thức liên kết quan hệ 1-n
+    public function post(){
```

Comment:
```suggestion
    public function posts() {
```
quan hệ many thì để số nhiều nha